share/wake/lib/system/job.wake

```diff
@@ -333,6 +333,11 @@ def jArray (fn: Result JValue Error => Result a Error) = match _
     Fail err = Fail err
     _ = failWithError "not an array"
 
+def getPath input =
+    require Pass elem = input
+    jField elem "path"
+    | jString
+
 export def mkJobCacheRunner ((Runner name score baseDoIt): Runner): Runner =
```

Comment:
dir_redirects is no longer correct. `"."` needs to be an absolute path to the workspace now.